strawberry_django/utils/typing.py

```diff
@@ -37,10 +39,13 @@
 _Type = TypeVar("_Type", bound="StrawberryType | type")
 
 TypeOrSequence: TypeAlias = Union[_T, Sequence[_T]]
+TypeOrMapping: TypeAlias = Union[_T, Mapping[str, _T]]
```

Comment:
I don't love that `annotate` needs a dict instead of a list, but I see no other way of specifying the annotation name in a case like this:
@strawberry_django.type(Milestone)
class MilestoneType(relay.Node):
    @strawberry_django.field(
        annotate={
            "_issues_count": lambda info: Count("issue")
        },
    )
    def issues_count(self) -> int:
        return self._issues_count  # type: ignore